### ISSUE TITLE
Can I move the pinokio folder to an external drive after install?

### ISSUE THREAD

**Issue by MarkBowenPiano** [pinokiocomputer/pinokio#203]
Hi there,

I'm using Pinokio on a Mac Mini M2 Pro to enable me to use Fooocus and so far everything has been really good.
Today I realised how much space the pinokio folder is taking up in my user folder so I decided to copy the folder to an external SSD drive (which is much much faster than the internal drive) and then created a symlink to that folder from within my user folder.

Starting up Fooocus now from within Pinokio takes a much longer time and in fact the Fooocus interface never ever shows. The whole process used to take at most around 10 seconds but now it takes at least 1-2 minutes just to start it all up.

I **can** access Fooocus from a web browser once it's all loaded _(haven't actually tried creating anything yet though)_ but I much prefer using it within the Pinokio app but it just never seems to load and just stays on the log screen where it shows you what is happening.

I see within Pinokio in the settings that you can set a path to use however I tried changing this to the external drive path and if I press save and come back into the settings again then the path is back to what it was originally, that of /users/myusername/pinokio where I would like it to instead be pointing to the external drive which is /Volumes/Work/pinokio

Is there therefore anything I did wrong with utilising a symlink and if so is there any way to have that folder located on my external drive and somehow use that from within the Pinokio app as at the moment it's not working the way it should :-(

Any help with this would be greatly appreciated.

Thank you,

Mark

**Comment by dotnetprofessional** [comment]
You're supposed to be able to if you just change the settings. It will copy the files and it will look good. However, when I did it, all the apps stopped working, the settings for each app weren't updated with the new file location. I'm now trying to do a clean install and now that's broken, can't get to the app install pages.

Recommendation, is to reinstall the app using your new location when installing. Once you have an empty install rename the new destination folder and copy your old folder to the name you just installed to. Then start the app and it 'should' work, but again you might need to modify the settings files if they are pointing to the wrong place. I think they are supposed to be relative.

**Comment by MarkBowenPiano** [comment]
> You're supposed to be able to if you just change the settings. It will copy the files and it will look good. However, when I did it, all the apps stopped working, the settings for each app weren't updated with the new file location. I'm now trying to do a clean install and now that's broken, can't get to the app install pages.
> 
> Recommendation, is to reinstall the app using your new location when installing. Once you have an empty install rename the new destination folder and copy your old folder to the name you just installed to. Then start the app and it 'should' work, but again you might need to modify the settings files if they are pointing to the wrong place. I think they are supposed to be relative.

Yep I had to reinstall it all again though I kept the models that had already downloaded and just placed them back after to save having to downloading them all again as not on the fastest internet connection here.
Does seem like you should just be able to update that setting but mine just would never stick and so the app wouldn't work when I did that.
Think there's definitely something not quite right somewhere but not sure what. All working now after a clean install though so everything is okay now. Thank you.

**Comment by AhmedRayyan** [comment]
i just figured that you can do it through it's config.json AppData file
if you are on windows you can locate it in : C:\Users\[UserName]\AppData\Roaming\Pinokio
add/change home key to your desired location, Like so : "home": "[DriveLetter]:\\[pinokioPath]"
